client/src/components/FeedbackForm.js

```diff
@@ -0,0 +1,238 @@
+import React from 'react';
+import { Grid, FormLabel, TextField, Typography, DialogTitle, DialogContentText, DialogContent } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles';
+import RubricBubble from './RubricBubble'
+
+const useStyles = makeStyles((theme) => ({
+    dialogTitleEmbed: {
+        fontSize:"1.25rem"
+    },
+    feedbackRadioContainer: {
+        display: "flex",
+    },
+    feedbackRadioButton: {
+        paddingLeft: "1.5rem",
+        paddingRight: "1.5rem",
+        paddingTop: ".5rem"
+    },
+    feedbackRadioLabel: {
+        alignSelf: "center",
+        width: "20%",
```

Comment:
as much as possible, i would avoid using percentage for CSS, unless you are absolutely certain that it will always be 20%, it'll get tricky if you ever want to maintain responsiveness down the line. 

No need to fix here, but just keep this in mind going forward. `Grid` and `Box` (any grid system, really) in MaterialUI would help you maintain this a lot better